script/latest_snapshot_version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+#
+# Find the latest snapshot version for the specified branch.
+#
+# Example usage: ./latest_snapshot_version.py 7.x
+
+BRANCH=$1
```

Comment:
**suggestion**: should we guard against `BRANCH` being empty and return an error ?